evap/staff/views.py

```diff
@@ -545,8 +580,17 @@ def evaluation_operation(request, semester_id):
                 plain_content=request.POST["email_plain_participant"],
                 html_content=request.POST["email_html_participant"],
             )
+        if request.POST.get("delete-previous-answers") == "on":
+            delete_previous_answers = True
```

Comment:
Couldn't we do `delete_previous_answers = request.POST.get("delete-previous-answers") == "on"` from the start? I think this would also fix the `bool | None` "issue", although I don't care so much about that

Reply:
We could do it like that, but then we would need to change the `apply` methods of all other operations to expect `False` instead of `None`. This is doable, but as all the other parameters use `None` I do not think it will contribute to readability.

```python
    @staticmethod
    def apply(
        request,
        evaluations,
        email_template=None,
        email_template_contributor=None,
        email_template_participant=None,
        delete_previous_answers=None,
    ):
        assert email_template_contributor is None
        assert email_template_participant is None
        assert delete_previous_answers is None
        
        # ....
```

 So I would argue for keeping it as it is now.

Reply:
I think the other `apply` methods requiring `None` is fine (not type-annotated in the current version, and subject to change anyway?), but we can still use `delete_previous_answers = request.POST.get("delete-previous-answers") == "on"` here.

Reply:
either way is fine - if we are divided on this, we can also leave it as is for now